README.md

```diff
@@ -7,3 +7,16 @@
 - Sprint 2
 - Sprint 3
 - Sprint 4
+
+## Coding conventions
+
+-`kebab-case` for doc names
+-`camelCase` for vars
+Do we have a preferred linter?
```

Comment:
ESLint + Prettier with recommended TypeScript, React, Prettier plugins.

Do we need to explicitly define all our coding conventions or do we want to leave that up to the linters and formatters?

Reply:
I don't think we need to explicitly define all our coding convention, but I think it is worth it to write down which linters we are gonna use, I'm cool with ESLint and Prettier with default configs

Reply:
Since we don't need to define our coding convention in the readme, should we remove this section?